src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/controller/advice/ControllerAdvice.kt

```diff
@@ -30,27 +28,13 @@ class ControllerAdvice {
     return ResponseEntity(ErrorResponse(status = 404, developerMessage = e.message), NOT_FOUND)
   }
 
-  @ExceptionHandler(MethodArgumentNotValidException::class)
```

Comment:
@Valid never used in project. In Javadocs `Exception to be thrown when validation on an argument annotated with @Valid fails.`

Reply:
no doubt copy/paste from another project